log/log.go

```diff
@@ -45,6 +46,9 @@ func (f levelFlag) Set(level string) error {
 // setSyslogFormatter is nil if the target architecture does not support syslog.
 var setSyslogFormatter func(string, string) error
 
+// setEventlogFormatter is nil if the target os does not support Eventlog (eg, is not Windows)
```

Comment:
Ok. As a non-native, I tend to confuse i.e. vs e.g., but now I've googled up some good memory tricks :)